apps/odyssey-stats/src/lib/init-sentry.ts

```diff
@@ -0,0 +1,7 @@
+export default function initSentry() {
+	const script = document.createElement( 'script' );
+	// Uses unique DNS specific to Odyssey Stats.
+	script.src = 'https://js.sentry-cdn.com/332d216a4d31158057fb5277cd48f438.min.js';
+	script.crossOrigin = 'anonymous';
```

Comment:
Would it be better if we set `async`  to true?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#async